python/activator/middleware_interface.py

```diff
@@ -1595,7 +1653,7 @@ def _get_safe_dataset_types(butler):
 
     @connect.retry(2, DATASTORE_EXCEPTIONS, wait=repo_retry)
```

Comment:
Sorry, one last question: does this need to be updated to account for `KafkaButlerWriter.transfer_outputs`? I assume it raises a completely different set of exceptions on e.g. network problems.